mplc/experiment.py

```diff
@@ -115,13 +120,14 @@ def run_experiment(self):
             logger.info(f"(Experiment {self.name}) Now starting repeat {repeat_index_str}")
 
             # Loop over scenarios in scenarios_list
-            for scenario_idx, scenario in enumerate(self.scenarios_list):
+            for scenario_idx, white_scenario in enumerate(self.scenarios_list):
```

Comment:
Fun color! I never heard about prefixing a variable with `white_...`, what does it correspond to?

Reply:
changed to blank_scenario. The one which is never ran, and which is copied.